_data/hosting_providers.json

```
@@ -939,6 +939,16 @@
     "reviewed": "",
     "note": ""
   },
+  {
+    "name": "ikoula",
+    "link": "https://www.ikoula.com/en/web-hosting",
+    "category": "partial",
+    "tutorial": "https://en-wiki.ikoula.com/en/Establishment_and_renewal_of_a_certificate_let%27s_encrypt",
+    "announcement": "[No need to find this]",
```

```suggestion
    "announcement": "",
```

"[No need to find this]" is an artifact left over from the spreadsheet days